docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md

```diff
@@ -1 +1,290 @@
-Mac specific instructions
\ No newline at end of file
+### Mac is no longer supported as of 15.10.
+Nothing in this document is an official recommendation for LORIS but is only intended to be helpful!
+
+## Important Notes: 
+- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported for "production" as of v15.10
+- These instructions were last tested on LORIS version v20.*
+
+# How to install a developer instance of LORIS on macOS
```

Comment:
Agreed with @ridz1208 that this should be moved to the `99` dev section so that it doesn't get mixed up with officially supported versions